docs/developer-guide/edp-workflow.md

```diff
@@ -1,30 +1,30 @@
-# EDP Project Rules. Working Process
+# KubeRocketCI Project Rules. Working Process
 
-This page contains the details on the project rules and working process for EDP team and contributors. Explore the main points about working with Gerrit, following the main commit flow, as well as the details about commit types and message below.
+This page contains the details on the project rules and working process for KubeRocketCI team and contributors. Explore the main points about working with GitHub, following the main commit flow, as well as the details about commit types and message below.
 
 ## Project Rules
 
 Before starting the development, please check the project rules:
 
-1. It is highly recommended to become familiar with the Gerrit flow. For details, please refer to the [Gerrit official documentation](https://gerrit-documentation.storage.googleapis.com/Documentation/3.4.1/index.html) and pay attention to the main points:
+1. It is highly recommended to become familiar with the GitHub flow. For details, please refer to the [GitHub official documentation](https://docs.github.com/en/get-started/using-github/github-flow) and pay attention to the main points:
 
-  a. Voting in Gerrit.
+  a. Creating pull requests in GitHub.
 
   b. Resolution of Merge Conflict.
 
   c. Comments resolution.
 
-  d. One Jira task should have one Merge Request (MR). If there are many changes within one MR, add the next patch set to the open MR by selecting the _Amend commit_ check box.
+  d. One Jira task should have one Merge Request (MR) if it doesn't change multiple operators. If there are many changes within one MR, amend the commit.
```

Comment:
are we using Jira in this document?
Maybe we're ready to move to GitHub release?

Reply:
fixed